threat_shield_dns.rst

```diff
@@ -0,0 +1,98 @@
+.. _threat_shield_dns-section:
+
+=================
+Threat Shield DNS
+=================
+
+Threat Shield DNS uses AdBlock which blocks any request to a domain that is considered malicious.
+The service can load community-maintained blocklist or use Enterprise feeds provided by Nethesis and Yoroi.
+
+While Threat Shield DNS is primarily configured from the command line, this chapter will guide you through its setup and usage.
+
+Enable Threat Shield DNS
+=========================
+
+To enable Threat Shield DNS, use the following commands: ::
+
+  echo '{"enabled": true, "zones": ["lan"]}' | /usr/libexec/rpcd/ns.threatshield call dns-edit-settings
+  uci commit adblock && service adblock restart
+
+This command enables Threat Shield DNS and applies it to the "lan" zone. 
+After enabling, all DNS requests from the configured zones are redirected to the firewall itself (ports 53 and 853 TCP/UDP). 
+Note that this port forwarding is not visible from the port forward page.
+
+If you want to filter other zones, replace "lan" with the desired zone name.
+
+.. note:: Please use Threat Shield DNS only if you are not already using the FlashStart service cause if used together, they may conflict.
```

Comment:
```suggestion
.. note:: Please use Threat Shield DNS only if you are not already using the FlashStart service because if used together, they may conflict.
```